ESP_SHT3x/ESP_SHT3x.ino

```diff
@@ -32,13 +99,33 @@ void printShtMeasurments(float temp, float hum) {
     Serial.println("Failed to read humidity");
   }
 }
+
+// Sets relay pins to high or low depending on measurements
+void relayPower(int pin,float current, float limit) {
+  if(current < limit) {
+    digitalWrite(pin, LOW);
+  } else {
+    digitalWrite(pin, HIGH);
+  }
+}
  
 void loop() {
+  // Checks and initializes sht31
   shtInit();
+
+  // Reads temperature and humidity from sht31 and stores it to appropriate variables
   float t = sht31.readTemperature();
   float h = sht31.readHumidity();
 
+  // Displays current readings
   printShtMeasurments(t, h);
```

Comment:
is this even a function? I'm not seeing it.

Reply:
It's on line 89. The function is only meant for testing purposes. It will be replaced by a function which makes a post to the rpi in a future branch.